packages/api-hooks/resources/events/index.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import { useQuery, UseQueryOptions } from "@tanstack/react-query"
+import { GetAllEvents } from './types'
+import { createAxiosClientWithAuth } from '../../lib/axios';
+
+const EVENTS_ROOT_PATH = '/events';
+
+
+const generateGetAllEventsWithAuth = (idToken: string) => {
+  console.log({ idToken })
```

Comment:
We might want to remove the console.log with idToken since it's conveying sensitive information.